esmvaltool/diag_scripts/arctic_ocean/utils.py

```diff
@@ -0,0 +1,324 @@
+# -*- coding: utf-8 -*-
+"""Part of the ESMValTool Arctic Ocean diagnostics.
+
+This module contains utility functions.
+"""
+import logging
+import os
+import shutil
+
+import cmocean.cm as cmo
+import matplotlib as mpl
+import matplotlib.cm as cm
+import numpy as np
+import pyproj
+import seawater as sw
+from cdo import Cdo
+
+from esmvaltool.diag_scripts.shared._base import (ProvenanceLogger)
+
+mpl.use('agg')
```

Comment:
```suggestion
```
can be removed, as esmvaltool uses an environmental variable to set this for all diagnostic scripts